package.json

```diff
@@ -16,7 +16,7 @@
   ],
   "dependencies": {
     "@antora/cli": "^1.1.1",
-    "@antora/site-generator-default": "^1.1.1"
+    "@antora/site-generator-default": "^2.0.0"
```

Comment:
Why was the Antora version bumped in this PR? This should, ideally, be in a separate PR.

Reply:
ok, sorry, dondn't know that

Reply:
All good. Just worth mentioning.